src/Dialect/ONNX/ONNXOps/ShapeHelper.hpp

```diff
@@ -352,8 +352,8 @@ using ONNXAtanOpShapeHelper = ONNXUnaryOpShapeHelper;
 using ONNXAtanhOpShapeHelper = ONNXUnaryOpShapeHelper;
 using ONNXBernoulliOpShapeHelper = ONNXUnaryOpShapeHelper;
 using ONNXBitwiseNotOpShapeHelper = ONNXUnaryOpShapeHelper;
-using ONNXCastLikeOpShapeHelper = ONNXUnaryOpShapeHelper;
 using ONNXCastOpShapeHelper = ONNXUnaryOpShapeHelper;
+using ONNXCastLikeOpShapeHelper = ONNXUnaryOpShapeHelper;
```

Comment:
Thanks, it feel safer to me to have it as a unary. 
Nit: alphabetically listed.